elasticache/variables.tf

```diff
@@ -25,7 +25,7 @@ variable "node_type" { default = "cache.t3.micro" }
 
 variable "node_count" {
   type        = number
-  default     = 1
```

Comment:
otherwise, I receive this error

```
module.stack.module.elasticache.aws_elasticache_replication_group.non_cluster_mode[0]: Creating...
╷
│ Error: error creating ElastiCache Replication Group (lh-flightpass-staging): InvalidParameterCombination: When using automatic failover, there must be at least 2 cache clusters in the replication group.
│ 	status code: 400, request id: d0e1c9bb-b820-4e8b-be54-550927b5431f
│
│   with module.stack.module.elasticache.aws_elasticache_replication_group.non_cluster_mode[0],
│   on .terraform/modules/stack/elasticache/elasticache.tf line 1, in resource "aws_elasticache_replication_group" "non_cluster_mode":
│    1: resource "aws_elasticache_replication_group" "non_cluster_mode" {
│
```

Reply:
I think for staging environments, we should only run 1 node to save costs, and disable automatic failover in such case.

Reply:
done